cmake: flash: Remove fake path to zephyr_flash_debug.py

galak opened the pull request:
We append a path to the FLASH_SCRIPT/DEBUG_SCRIPT that was bogus and not
really needed by zephyr_flash_debug.py.  So lets remove it since its
just confusing.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

galak committed:
cmake: flash: Remove fake path to zephyr_flash_debug.py

We append a path to the FLASH_SCRIPT/DEBUG_SCRIPT that was bogus and not
really needed by zephyr_flash_debug.py.  So lets remove it since its
just confusing.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

mbolivar commented:
It's not bogus -- it was needed formerly when the Python script would fall back on the shell script if it didn't know how to do something.

I agree that it's vestigial now, but do you mind also touching up the "shell_script_full" variable in zephyr_flash_debug.py to finish the job?

galak committed:
scripts: zephyr_flash_debug: Cleanup the script argument usage

The script argument isn't really a path, so stop assuming that it is
one.  We still use the shell script name at this point, but there isn't
any actual shell script in the system.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

galak commented:
> It's not bogus -- it was needed formerly when the Python script would fall back on the shell script if it didn't know how to do something.
> 
> I agree that it's vestigial now, but do you mind also touching up the "shell_script_full" variable in zephyr_flash_debug.py to finish the job?

Add a commit to cleanup zephyr_flash_debug.py, see if that looks good.

mbolivar reviewed (approved):
Looking good, thanks for that.
